common/api/core-common.api.md

```diff
@@ -7344,7 +7342,7 @@ export class QPoint3dBufferBuilder {
 
 // @public
 export class QPoint3dList {
-    [Symbol.iterator](): IterableIterator<QPoint3d>;
+    [Symbol.iterator](): ArrayIterator<QPoint3d>;

```

Comment:
Digging through the TypeScript 5.6 documentation and their .d.ts files, I can't seem to find an explanation of what `ArrayIterator`, `MapIterator`, et al actually **are**. Do you have a link to more information?

Reply:
Ok, all it does is define return type in terms of `BuiltinIteratorReturn`.